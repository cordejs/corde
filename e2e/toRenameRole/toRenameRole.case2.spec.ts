import { runCLI } from "../cliRunner";
import Utils from "../testUtils";

it("tests should be not be sucessfull", async () => {
  const command = Utils.buildCommandWithConfigPath("toRenameRole", "bot_case2.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(mockProcess).toBeCalledWith(1);
  expect(stdout).toMatchSnapshot();
});
