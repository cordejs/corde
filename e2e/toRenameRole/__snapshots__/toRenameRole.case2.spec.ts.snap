// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests should be not be sucessfull 1`] = `
"node test --config ./e2e/corde.config.ts --files ./e2e/toRenameRole/__cordeTest__/bot_case2.spec.ts

 FAIL   e2e\\\\toRenameRole\\\\__cordeTest__\\\\bot_case2.spec.ts   100ms
   x should fail when renaming a role   100ms
    
    expected: role 'old-role-name' to be renamed to batata
    received: name was not changed

    at toRenameRole e2e\\\\toRenameRole\\\\__cordeTest__\\\\bot_case2.spec.ts:8:49
    at _internalTest (lib\\\\src\\\\closures\\\\test.js:29:13)
    at lib\\\\src\\\\closures\\\\test.js:52:65

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
